ci/azp-private.yml

```diff
@@ -0,0 +1,10 @@
+# Private CI trigger
```

Comment:
`# Private CI trigger. Used to run tooling that can't currently be shared publicly.`